quickjs-libc.c

```diff
@@ -76,10 +78,12 @@ typedef sig_t sighandler_t;
 extern char **environ;
 #endif
 
+#endif /* _WIN32 */
+
+#if !(defined(_WIN32) || defined(__wasi__))
```

Comment:
I find that hard to read, personally - it's easy to miss the negation. Can I suggest writing it like this?
```suggestion
#if !defined(_WIN32) && !defined(__wasi__)
```